decentraland/unity-explorer
# Chore: analytics disable chat message

## What does this PR change?

- changed namings performance analytics for memory
- changed memory budget to use Total Used Memory instead of System one
- add a flush on realm change
- disabled chat message
- converted Nanosec to Ms for total time of performance report


## How to test the changes?


1. Launch the explorer
2. Verify happy path is fine

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
